packages/playground/src/weblets/tf_nextcloud.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <weblet-layout
+    ref="layout"
+    :cpu="solution?.cpu"
+    :memory="solution?.memory"
+    :disk="(solution?.disk ?? 0) + rootFilesystemSize"
+    :ipv4="ipv4"
+    :certified="certified"
+    :dedicated="dedicated"
+    title-image="images/icons/nextcloud.png"
+  >
+    <template #title>Deploy a Nextcloud All-in-One Instance </template>
+    <input-validator
+      :value="name"
+      :rules="[
+        validators.required('Name is required.'),
+        validators.isLowercase('Name should consist of lowercase letters only.'),
+        validators.isAlphanumeric('Name should consist of alphabets & numbers only.'),
+        name => validators.isAlpha('Name must start with alphabet char.')(name[0]),
+        validators.minLength('Name must be at least 2 characters.', 2),
+        validators.maxLength('Name cannot exceed 15 characters.', 15),
+      ]"
+      #="{ props }"
+    >
+      <input-tooltip tooltip="Instance name.">
+        <v-text-field label="Name" v-model="name" v-bind="props" />
+      </input-tooltip>
+    </input-validator>
+
+    <input-validator
+      :value="domain"
+      :rules="[
+        validators.required('Domain is required.'),
+        validators.pattern('Please provide a valid domain.', {
+          pattern: /^\b((?=[a-z0-9-]{1,63}\.)(xn--)?[a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,63}\b$/,
+        }),
+      ]"
+      #="{ props }"
+    >
+      <input-tooltip tooltip="Domain name.">
+        <v-text-field label="Domain" v-model="domain" v-bind="props" />
+      </input-tooltip>
+    </input-validator>
+
+    <v-alert type="warning" variant="tonal" class="mb-6">
+      <p :style="{ maxWidth: '880px' }">
+        You will need to add an A record (Host: "@", Value: &lt;VM_IP_Address&gt;) for this domain after deployment.
+        Otherwise, you won't be able to access the Nextcloud AIO instance using this domain.
+      </p>
+
+      <p class="font-weight-bold mt-4">
+        For more information, read the
+        <a
+          target="_blank"
+          href="https://www.manual.grid.tf/terraform/advanced/terraform_nextcloud_aio.html"
```

Comment:
I think the href should be linked to a page under the Playground directory in the info grid, not Terraform. Also, it should redirect to the domain section, not the whole page.

Reply:
Noted. New documentation [here](https://github.com/threefoldtech/info_grid/pull/240).